requirements/test.txt

```diff
@@ -10,3 +10,4 @@ pandas
 lightning
 lightning-cloud == 0.5.70 # Must be pinned to ensure compatibility
 zstd
+numpy < 2.0
```

Comment:
Why this though?
I believe a new PR was merged recently which added support for numpy>=2!